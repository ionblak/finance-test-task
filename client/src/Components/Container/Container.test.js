import { render } from '@testing-library/react';
import Container from './Container';

describe('Container', () => {
  it('Container should render properly', () => {
    const { container } = render(<Container />);
    expect(container).toMatchSnapshot();
  });
});
