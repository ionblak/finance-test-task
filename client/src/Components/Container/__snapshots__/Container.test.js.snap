// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Container should render properly 1`] = `
<div>
  <div
    class="container"
  />
</div>
`;
